package.json

```diff
@@ -2,60 +2,64 @@
     "name": "escodegen",
     "description": "ECMAScript code generator",
     "homepage": "http://github.com/estools/escodegen",
+    "bugs": "http://github.com/estools/escodegen/issues",
     "main": "escodegen.js",
     "bin": {
         "esgenerate": "./bin/esgenerate.js",
         "escodegen": "./bin/escodegen.js"
     },
+    "keywords": [
+        "code",
+        "generation"
+    ],
     "files": [
         "LICENSE.BSD",
         "README.md",
         "bin",
         "escodegen.js",
+        "escodegen.browser.min.js",
+        "escodegen.browser.js",
         "package.json"
     ],
     "version": "1.14.3",
     "engines": {
         "node": ">=4.0"
```

Comment:
`"node": ">=10.0"` should be here

Reply:
Per https://github.com/estools/esquery/pull/96#issuecomment-608642544 and context, @michaelficarra was inclined for the project to keep `engines` and only bump if the project is known to fail. I was planning on also adding a PR for babel support on top of this as should allow that backward support (but had been informed there was too much in this PR originally to include the Rollup/Babel routine). So I can bump `engines` if desired, or add Babel to provide backward-support, but if the latter, I presume Babel is still desired in another PR.

Reply:
Btw, I've rebased on master, and also noted in my intro comment that I've also added these changes on top of the previously mentioned ones:

- npm: Restore optionator to a regular dep. (used in published binary file)
- npm: Drop unused semver, minimist

Reply:
source-map 0.7.x drops support for Node < 8.

Reply:
Ok, but `source-map` has been a part of `optionalDependencies`.